data/data/ovirt/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
+
+approvers:
```

Comment:
make sure all the OWNERS files look like https://github.com/openshift/installer/blob/master/data/data/aws/OWNERS

both approvers and reviewers should exist...

Reply:
Got it, done